src/Sushi.Mediakiwi/AppCentre/UI/NoteList.cs

```diff
@@ -21,10 +21,13 @@ public NoteList()
             ListLoad += NoteList_ListLoad;
         }
 
+
+
         async Task NoteList_ListLoad(ComponentListEventArgs e)
         {
             int lastError = wim.CurrentVisitor.Data["last.error"].ParseInt().GetValueOrDefault();
-            var note = await Mediakiwi.Data.Notification.SelectOneAsync(lastError).ConfigureAwait(false);
+            var repository = new Mediakiwi.Data.Repositories.Sql.NotificationRepository();
```

Comment:
Shouldn't this repository be fetched from DI ?

Reply:
that would break backwards compatability (it would require setting up the DI when updating to this new MK version)

Reply:
I see, but you could use this line as fallback when no INotificationRepository has been added to DI, that way the list can function for as well Elastic and SQL notifications right ?